sdks/ios/runtime/runtime.m

```diff
@@ -355,6 +359,20 @@
 	return result;
 }
 
+//
+// Returns the geopolitical region ID of the local timezone.
+// Previously we just provided the data to TimeZoneInfo.MonoTouch.cs and that
```

Comment:
nit: a few years from now readers of this code won't care what we used to do a long time ago (that is what git history is for), I'd remove this sentence.